### PR TITLE
Basic UI : Fixes #120

### LINKED ISSUE #120: Build UI
### Expected results
An interface with systematic buttons and content

### Actual results
image

### Steps to reproduce the issue
- Add buttons and other required items to make the game look systematic

## Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by YashKumarVerma**
This pull request is related to issue #120  

I modified the labyrinth, you can test my changes here:
http://rawgit.com/yashkumarverma/labyrinth/master/index.html

The problem I want to solve is 
- We have 
image
and this doesn't look good.

My solution for the problem is 
## Statistics
image

## Control Your Character
image
image

Credits 👍 
https://github.com/Dogfalo/materialize

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -0,0 +1 @@
+/*Add custom CSS for Game here*/
```
Comment (niccokunzmann) [inline css/custom.css]: ok!

File: css/fonts.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@font-face {
```
Comment (niccokunzmann) [inline css/fonts.css]: This file seems ok, too... did you write it or is it from somewhere else? Then, we should include the license.

File: css/materialize.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*!
+ * Materialize v0.100.2 (http://materializecss.com)
+ * Copyright 2014-2017 Materialize
+ * MIT License (https://raw.githubusercontent.com/Dogfalo/materialize/master/LICENSE)
```
Comment (niccokunzmann) [inline css/materialize.css]: great, the license is here... all fine with this file.

File: css/fonts.css (hunk 2 above)
Comment (YashKumarVerma) [reply css/fonts.css]: https://google.github.io/material-design-icons/#icon-font-for-the-web

Comment (YashKumarVerma) [reply css/fonts.css]: https://github.com/google/material-design-icons

Comment (niccokunzmann) [reply css/fonts.css]: Ok. Could you put that link as a comment into the first lines of the file?

Comment (niccokunzmann) [reply css/fonts.css]: It does not require a license, I hope, because it is so small.

**Comment by YashKumarVerma** [comment]
@niccokunzmann  Done :smile

File: css/fonts.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/*
+Credits
+https://google.github.io/material-design-icons/#icon-font-for-the-web
```
Comment (niccokunzmann) [inline css/fonts.css]: thanks!

File: index.html [hunk 5]
```diff
@@ -2,27 +2,65 @@
...
+<!-- Data Card -->
+<div class="row">
+    <div class="col m4 s8">
+      <div class="card blue-grey darken-1">
+        <div class="card-content white-text">
+          <span class="card-title">Statistics</span>
+          <p>Player Moved : <span id="playerMoves">0</span> Times</p>
+        </div>
+        <div class="card-action">
+          <a href="http://github.com/yashkumarverma/">UI by Yash Kumar Verma</a>
```
Comment (niccokunzmann) [inline index.html]: This is nothing for this pull request, but we could create a credits.html file where we could mention many more people for different tasks.

**Comment by niccokunzmann** [comment]
Thanks a lot.

**Pull request merged**